packages/accounts/tsconfig.declarations.json

```diff
@@ -6,5 +6,5 @@
         "outDir": "./dist/types"
     },
     "extends": "./tsconfig.json",
-    "include": ["src/index.ts", "src/types"]
+    "include": ["../build-scripts/build-time-constants.d.ts", "src/index.ts"]
```

Comment:
I'm so confused, wasn't this done in the previous PR already? Is this a bad rebase from Graphite post-auto-merge?

Reply:
I think the github diff is messed up here - before state on main doesn't match at all: https://github.com/anza-xyz/kit/blob/main/packages/accounts/tsconfig.declarations.json

Maybe worth a manual rebase + push?